dev/com.ibm.ws.jsp/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -20,6 +20,22 @@
             id="disableJspRuntimeCompilation" required="false" type="Boolean" default="false"/>
         <AD description="%extendedDocumentRoot" name="%extendedDocumentRoot.name"
             id="extendedDocumentRoot" required="false" type="String" default="" /> 
+        <AD description="%javaSourceLevel" name="%javaSourceLevel.name"
+            id="javaSourceLevel" required="false" type="String">
+            <Option label="8" value="8"/>
+            <Option label="1.8" value="1.8"/>
+            <Option label="11" value="11"/>
+            <Option label="12" value="12"/>
+            <Option label="13" value="13"/>
+            <Option label="14" value="14"/>
+            <Option label="15" value="15"/>
+            <Option label="16" value="16"/>
+            <Option label="17" value="17"/>
+            <Option label="18" value="18"/>
+            <Option label="19" value="19"/>
+            <Option label="20" value="20"/>
+            <Option label="21" value="21"/>
```

Comment:
21 should be added later as it's not yet supported? Latest is 20: 

https://github.com/eclipse-jdt/eclipse.jdt.core/blob/3ba09a9bc92458dd4a018dbfa8dd7fc9402ce7c3/org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/impl/CompilerOptions.java#L244

Reply:
Issue: `https://github.com/eclipse-jdt/eclipse.jdt.core/issues/890`

Reply:
Follow up: Should we support intermediary releases?

Reply:
Alasdair said to only support the LTS releases.